lib/mixlib/shellout/unix.rb

```diff
@@ -304,6 +316,10 @@ def read_process_status_to_buffer
         open_pipes.delete(child_process_status)
       end
 
+      def cgroupv2_available?
+        !File.read("/proc/mounts")[%r{^cgroup2 /sys/fs/cgroup}].nil?
```

Comment:
`File.read("/proc/mounts").match?(/^cgroup2 /sys/fs/cgroup/)` would remove the `.nil?` call and the `!` negation... unless you're depending on `MatchData` vs. just checking for presence.